readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://MykhailoLoniak.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://MykhailoLoniak.github.io/layout_moyo-header/report/html_report/)
```

Comment:
These links should be into Pull request description 
![image](https://github.com/mate-academy/layout_moyo-header/assets/70009205/f4b0af84-6add-45b3-9eb4-8c694892a4a2)